### DISCUSSION TITLE
Add style props for each side of the border

### DISCUSSION THREAD

**Discussion by ItsMalma** [mantinedev/mantine#6486] [Feature requests]
I'm very happy with the addition of the `bd` style prop in the mantine to add a border, but sometimes I just want to add a border on one side only (or not on all sides).

I plan to make a PR to add new props for borders (`bdt`, `bdb`, `bde`, etc) such as margin props (`mt`, `mb`, `me`, etc).

However, I wonder if there is a better solution for this? Or maybe just stick to the CSS way?

**Comment by Julhol-droid** [comment]
Any progress on that? That would be really helpfull

**Comment by tysbrown** [comment]
I also feel this is missing from the Style API. Did you end up working on this @ItsMalma I'm happy to pick it up.
